### PR TITLE
checkpoint 01

### PR DESCRIPTION
**Pull request by AskForTheWolf**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
   <head>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Checkpoint 01, The Wolf</title>
   </head>
   <body>
-    <header>
+    <div class='header'>
       <h1>The Header</h1>
-    </header>
+    </div>
```
Comment (superbuggy) [inline index.html]: a `<header>` is a semantic element but a `<div>` is not a semantic element

File: index.html [hunk 2]
```diff
@@ -1,19 +1,21 @@
 <!doctype html>
 <html>
   <head>
+    <link rel="stylesheet" type="text/css" href="styles.css">
```
Comment (superbuggy) [inline index.html]: 👌

File: styles.css [hunk 3]
```diff
@@ -35,7 +37,35 @@ img {
...
+
+  }
+  footer{
+    order: 5;
+  }
+
+}
+
+
+header, footer, .main, aside > * {
```
Comment (superbuggy) [inline styles.css]: exactly the right kind of thinking! you have to add `> *` to each however.

```css
header > *, footer > *, .main > *, aside > * {
}
```

File: workflow.txt [hunk 4]
```diff
@@ -1,10 +1,7 @@
...
-git commit -m ""
-git fetch --all
 git add .
-git pull upstream master
-git merge my-solution
-fork on github
-git checkout master
+git commit -m ""
 git push origin my-solution
+git merge my-solution
```
Comment (superbuggy) [inline workflow.txt]: don't need the last line (LINE 7), but everything else is perfect

File: styles.css [hunk 5]
```diff
@@ -35,7 +37,35 @@ img {
...
+    order: 3;
+  }
+  aside{
+    order: 4;
+
+  }
+  footer{
+    order: 5;
+  }
+
```
Comment (superbuggy) [inline styles.css]: 👌

File: styles.css [hunk 6]
```diff
@@ -23,6 +23,8 @@ aside {
 }
```
Comment (superbuggy) [inline styles.css]: 👍

File: index.html [hunk 7]
```diff
@@ -1,19 +1,21 @@
...
+    </div>
     <aside>
       <h2>A Side Bar</h2>
     </aside>
     <div class='main'>
       <h2>The Main Content</h2>
       <div class='left'>
       	This is the left side
-      	<img src='https://jesse.sh/img/me.jpg'>
+      	<img src="https://jesse.sh/img/me.jpg">
```
Comment (superbuggy) [inline index.html]: missing an alt tag

**Comment by AskForTheWolf** [comment]
Thanks!

**Pull request closed without merging**